### PR TITLE
HaplotypeCallerIntegrationTest: add a boolean toggle to update the expected outputs for all exact-match-based tests

### PR DESCRIPTION
**Pull request by droazen**

### CODE REVIEW

**Comment by droazen** [comment]
@ldgauthier please review

**Comment by nh13** [comment]
Thanks @droazen, this will be super helpful

**Comment by droazen** [comment]
@ldgauthier Good point! I'll add a test that fails if the boolean toggle is left turned on.

**Comment by droazen** [comment]
Added a new test to assert that the toggle is off -- back to you @ldgauthier

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java [hunk 1]
```diff
@@ -42,8 +42,22 @@
...
+    // diffs in the new expected outputs in git to confirm that they are consistent with expectations.
+    public static final boolean UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS = false;
+
     public static final String TEST_FILES_DIR = toolsTestDir + "haplotypecaller/";
 
+    /*
+     * Make sure that someone didn't leave the UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS toggle turned on
+     */
+    @Test
+    public void assertThatExpectedOutputUpdateToggleIsDisabled() {
```
Comment (ldgauthier) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java]: So elegantly simple.

**Pull request merged**